### PR TITLE
update: update Kotlin/JS docs

### PR DESCRIPTION
**Pull request by katia-energizer**
Updated Kotlin/JS docs as agreed with Egor Tolstoy and Nastya Kapanina.

### CODE REVIEW

File: docs/topics/js/js-project-setup.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-[//]: # (title: Set up a Kotlin/JS project)
+[//]: # (title: Understand a Kotlin/JS project)
```
Comment (koshachy) [inline docs/topics/js/js-project-setup.md:1]: This page isn't about Understanding the project.
It shows how to set up JS-related Gradle props. I'd like you to rollback the changes (or start a discussion on why it should be named "Understand").

**Review by koshachy — CHANGES_REQUESTED** [review]

File: docs/topics/js/js-project-setup.md (hunk 1 above)
Comment (katia-energizer) [reply docs/topics/js/js-project-setup.md:1]: We can keep `Set up`.

**Review by koshachy — APPROVED** [review]
Nice job.

Don't forget to delete pics that are used in tutorials (multiplatform-full-stack-app and multiplatform-library).

File: docs/topics/js/js-project-setup.md (hunk 1 above)
Comment (katia-energizer) [reply docs/topics/js/js-project-setup.md]: Done ✅

**Review by danil-pavlov — APPROVED** [review]

**Pull request merged**